.github/workflows/create_release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create GitHub Release for Extension Package
+
+on:
+    workflow_dispatch:
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Install Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '16.x'
+
+            - name: Install dependencies
+              run: npm install
+
+            - name: Build VSIX Package
+              run: npx vsce package
+
+            - name: Create release
+              id: create_release
+              uses: actions/github-script@v6
+              with:
+                  script: |
+                      const fs = require('fs');
+                      const packageVersion = require('./package.json').version;
+                      const response = await github.rest.repos.createRelease({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        tag_name: `v${packageVersion}`,
+                        name: `Release ${packageVersion}`,
+                        body: `Generating release for version ${packageVersion}`
+                      });
+                      return response.data.id;
+            - name: Attach VSIX package to release
```

Comment:
nit nit: Missing newline since other steps are delineated by single line space in between.

Reply:
Haha I was just counting on the code formatting to fix that kind of stuff for me! I guess not. 😆 Good catch!